src/Arduino.cxx

```diff
@@ -0,0 +1,126 @@
+/*
+ *  Arduino.cxx
+ *  Copyright 2020 ItJustWorkTM
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#include <ardpolyfills\Arduino.hxx>
+#include <ardpolyfills\BoardData.hxx>
+#include <ardrivo/include/ardrivo/entrypoint.hxx>
```

Comment:
You shouldn't have to do this messy include; moving this file to the ardrivo module (where it belongs) will let you have cleaner `#include` directives